parsers/TR.py

```diff
@@ -37,6 +36,16 @@
 }
 
 
+def _str_to_datetime(date_string: str) -> datetime:
+    """
+    Converts string received into datetime format.
+    String received is almost in isoformat, just missing : in the timezone
+    """
+    return datetime.fromisoformat(date_string[:-2] + ":" + date_string[-2:]).replace(
+        tzinfo=TR_TZ
+    )
+
+
```

Comment:
I would wrap this in a try except cause so we can get some clear error logging if this fails (in case they "fix" the format). I would also remove the tzinfo replacement for the same reason. Since they include it in the data we should use it (in case it changes in the future).